lxqt-leave/CMakeLists.txt

```diff
@@ -77,3 +77,7 @@ install(FILES
     DESTINATION "${CMAKE_INSTALL_MANDIR}/man1"
     COMPONENT Runtime
 )
+install(DIRECTORY
+        resources/actions
+        DESTINATION "${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/scalable/"
```

Comment:
Please, use here the the `CMAKE_INSTALL_DATAROOTDIR` from the [`GNUInstallDirs` module](https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html)